src/sentry/sentry_metrics/indexer/id_generator.py

```diff
@@ -1,13 +1,14 @@
 import random
 import time
 
-_VERSION_BITS = 4
+_ZERO_BIT = 1
+_VERSION_BITS = 3
```

Comment:
Should we keep _VERSION_BITS as 4 and reduce the bits somewhere else? With 4 bits we can have 16 versions. With 3 we can have 8 only. Having more version bits would allow us to migrate more often if needed.

Reply:
This is annoying but technically there are 4 bits in the version. This just tries to force the versions to be "even" numbers. This is because the static string indexer is already a 64 bit unsigned integer with 1 in the leading bit. So that would technically make this version 2.

Reply:
@nikhars Updated to leave the version as 4 bits but enforce it's an even number. Let me know if this is clearer.